### PR TITLE
fix(test-runner): support passing slowMo option

### LINKED ISSUE #6984: slowMo does not exist in type 'Fixtures<{}, {}, PlaywrightTestOptions, PlaywrightWorkerOptions>'
Hi!
I'm trying to set up config (TypeScript) for test runner (1.12.0) as described here - https://playwright.dev/docs/test-configuration
But got the error for `slowMo` property - 
`Object literal may only specify known properties, and 'slowMo' does not exist in type 'Fixtures<{}, {}, PlaywrightTestOptions, PlaywrightWorkerOptions>'`

Screenshot from 2021-06-09 12-38-08

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes #6984
Revers #6967

### CODE REVIEW

File: src/test/cli.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ function overridesFromOptions(options: { [key: string]: any }): Config {
     forbidOnly: options.forbidOnly ? true : undefined,
     globalTimeout: isDebuggerAttached ? 0 : (options.globalTimeout ? parseInt(options.globalTimeout, 10) : undefined),
     grep: options.grep ? forceRegExp(options.grep) : undefined,
-    maxFailures: options.x ? 1 : (options.maxFailures ? parseInt(options.maxFailures, 10) : undefined),
+    maxFailures: options.maxFailures ? 1 : (options.maxFailures ? parseInt(options.maxFailures, 10) : undefined),
```
Comment (pavelfeldman) [inline src/test/cli.ts:149]: What is happening here?

File: src/test/index.ts [hunk 2]
```diff
@@ -87,6 +91,8 @@ export const test = _baseTest.extend<PlaywrightTestArgs & PlaywrightTestOptions,
       options.bypassCSP = bypassCSP;
     if (colorScheme !== undefined)
       options.colorScheme = colorScheme;
+    if (reducedMotion !== undefined)
```
Comment (pavelfeldman) [inline src/test/index.ts:94]: What does this all code do? :)

File: types/test.d.ts [hunk 3]
```diff
@@ -997,6 +1011,12 @@ export type PlaywrightTestOptions = {
    */
   colorScheme: ColorScheme | undefined;
 
+  /**
```
Comment (pavelfeldman) [inline types/test.d.ts:1014]: I thought those would go through as is, now I can see that we wrap each value :/

File: src/test/index.ts (hunk 2 above)
Comment (mxschmitt) [reply src/test/index.ts:94]: Pass it from the worker options to the context options.

File: types/test.d.ts (hunk 3 above)
Comment (mxschmitt) [reply types/test.d.ts:1014]: Yeah :/ lets rework it in a follow-up.

File: src/test/index.ts (hunk 2 above)
Comment (aslushnikov) [reply src/test/index.ts:94]: is this a drive-by?

Comment (mxschmitt) [reply src/test/index.ts:94]: It is 👍

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**